src/Symfony/Component/Yaml/README.md

```diff
@@ -1,21 +1,33 @@
 Yaml Component
 ==============
 
-YAML implements most of the YAML 1.2 specification.
+[![Package version](https://img.shields.io/packagist/v/symfony/yaml.svg)](https://packagist.org/packages/symfony/yaml)
```

Comment:
As it's a policy on the repo I guess there is not much that can be done here. Still, do you have a means to display the latest composer version available automatically without the badge? Otherwise would a link to packagist be acceptable?